src/Expensify.js

```diff
@@ -202,7 +206,7 @@ class Expensify extends PureComponent {
                 )}
 
                 <NavigationRoot
-                    onReady={this.setNavigationReady}
+                    onReady={this.setNavigationReady()}
```

Comment:
Shouldn't immediately execute this function

```suggestion
                    onReady={this.setNavigationReady}
```